beelzebub-chart/templates/service.yaml

```
@@ -7,9 +7,11 @@ metadata:
 spec:
   type: {{ .Values.service.type }}
   ports:
-    - port: {{ .Values.service.port }}
-      targetPort: http
+    {{- range $value := .Values.beelzebubServiceConfigs }}
+    - port: {{ trimPrefix ":" $value.config.address }}
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)